docs/kubernetes/kctlr-use-bigip-k8s.rst

```diff
@@ -67,31 +76,63 @@ Create a BIG-IP VXLAN tunnel
 
 .. _k8s-flannel-setup:
 
-Configure Flannel for Kubernetes
---------------------------------
-In order to deploy Flannel in your Kubernetes cluster, use the `kube-flannel.yml`_ file provided by Flannel.
-You can manually create this or use it as a `network addon`_ during kubeadm's setup. This file deploys Flannel
-on each Kubernetes Node, configuring the VXLAN network.
+Deploy flannel for Kubernetes
+-----------------------------
+
+#. (Optional) Edit the flannel `kube-flannel manifest`_.
+
+   - ``Backend``: Defaults to ``vxlan``; see `Backends <https://github.com/coreos/flannel/blob/master/Documentation/backends.md>`_ for additional information.
+   - ``Network``: Should match the Pod network CIDR. Flannel will populate this information automatically.
+
+.. todo:: Verify with @sjberman whether the above statue is correct.
+
+#. Use :command:`kubectl apply` to deploy the flannel manifest.
+
+   .. code-block:: console
+
+      kubectl apply -f kube-flannel.yaml --namespace=kube-system
+
+   \
+
+   .. tip::
+
+      You can also deploy flannel as a `network addon`_ when setting up ``kubeadm``. See :k8sdocs:`Installing a Pod network <setup/independent/create-cluster-kubeadm/#pod-network>` for instructions.
```

Comment:
I think the preferred way to deploy the yaml file is via a network addon for kubeadm. We should mention this first, and have an aside that says you could also manually deploy.